DEPRECATIONS.md

```diff
@@ -0,0 +1,67 @@
+# Deprecations & Breaking Changes in KEDA
+
+This document describes how KEDA handles deprecations and breaking changes and should provide clarity to KEDA end-users on what to expect when upgrading to a new version of KEDA.
+
+If you are interested in the current open deprecations, we recommend checking our [GitHub Discussions](https://github.com/kedacore/keda/discussions/categories/deprecations) or the open breaking changes](https://github.com/kedacore/keda/issues?q=is%3Aopen+label%3Abreaking-change+sort%3Aupdated-desc).
+
+## Our versioning strategy
+
+KEDA follows the [Semantic Versioning](https://semver.org/) strategy and can only introduce breaking changes in new major versions.
+
+However, depending on the nature of the change, we may introduce deprecations in existing minor versions. These changes always must be backwards-compatible.
+
+Depending on the area of the change, we might not need to wait for the next major version of KEDA to introduce a breaking change.
+
+### KEDA runtime
+
+The KEDA runtime, which consists of the operator and metric server, is not allowed to make breaking changes unless the major version is incremented.
+
+There are no exceptions to this rule.
+
+### Custom Resource Definitions (CRDs)
+
+Our custom resource definitions (CRDs) have a seperate versioning from the KEDA runtime given they have an `apiVersion`.
```

Comment:
I'd prefer if we link kubernetes policy here. alpha/beta/stable

Reply:
What about something like this?

```suggestion
Our custom resource definitions (CRDs) have a separate versioning scheme than the KEDA runtime given they have an `apiVersion`. For this, we follow the [official Kubernetes API versioning policy](https://kubernetes.io/docs/reference/using-api/#api-versioning).
```